src/components/EditCoursePage/EditCoursePage.test.jsx

```diff
@@ -64,6 +65,7 @@ describe('EditCoursePage', () => {
           end: defaultEnd,
           upgrade_deadline_override: '2019-05-10T00:00:00Z',
           variant_id: null,
+          restriction_type: restrictionType,
```

Comment:
do we need this value across all test data? Shouldn't coalesce handle the missing data?

Reply:
This blob of data is a "mock" for the network response. Since the network response only has `null`, `custom-b2c` and `custom-b2b-enterprise` as possible values, I am testing here with both a null and string response. This is the string version and the course run below is the null version.

Reply:
Got it.